### PR TITLE
Homework Tools #2: git, vim, compilers

### PR DESCRIPTION
**Pull request by Tomonike**
Add homework tools 2 : git, vim, compilers

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,3 @@
 all: src/*.cpp inc/*.hpp
-	g++ -std=c++17 src/*.cpp -o main
+	g++ -std=c++17 src/*.cpp -Werror -I./inc -I./src -o main
```
Comment (Pawellos) [inline Makefile]: You could add more flags. Like -Wall -Wextra -pedantic -Wconversion

File: inc/functions.hpp [hunk 2]
```diff
@@ -3,4 +3,6 @@
 void print(std::vector<int> const &);
 void sort(std::vector<int> &);
 int min(std::vector<int> const &);
+void reverse(std::vector<int> &);
+
```
Comment (Marcinati) [inline inc/functions.hpp]: One empty line should be enough.

File: Makefile (hunk 1 above)
Comment (Kasperice) [reply Makefile]: Yes, when you apply @Pawellos comments, you will see that this code is not correct. I mean something in `print` function in main.

**Review by Kasperice — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (Tomonike) [reply Makefile]: Ok. I change Makefile and functions `print`

File: inc/functions.hpp (hunk 2 above)
Comment (Tomonike) [reply inc/functions.hpp]: Deleted empty line.

**Comment by ziobron** [comment]
Great, all point :)

+4 XP Narzędzia#2 – vimtutor
+5 XP Narzędzia#2 – git - konflikty
+4 XP Narzędzia#2 – flagi kompilacji
+3 XP Narzędzia#2 – git - cherrypick
+8 XP delivery before 07.06.2020

{ "requirement_ids": [1119, 1999, 1119, 2000, 1119, 2001, 1119, 2002], "points": [2, 4, 2, 5, 2, 4, 2, 3] }

**Pull request closed without merging**